modules/lib/core/index.d.ts

```diff
@@ -356,9 +484,9 @@ export type QueryDsl = {
     exists: ExistsDslExpression;
 };
 
-type SortDirection = 'ASC' | 'DESC';
+export type SortDirection = 'ASC' | 'DESC';
```

Comment:
Do we actually need these types to be imported? They are only used inside `FieldSortDsl`, so why not just accessing them via `FieldSortDsl['direction']` or `GeoDistanceSortDsl['unit']`, like we do with some other types?

Reply:
Yes, they are used in our other type packages.

Reply:
If we want to limit the number of exports and imports, I think we should go for namespaces instead. Much nicer syntax and autocompletion UI in IDE's.

Reply:
It looks like these are simply re-exports from two other packages. While namespaces are useful, that’s not the issue that they'll fix, as we’re not adding these types to the global scope.

My main concern is whether we really need to export these. It’s easier to add them later if needed, but removing them could be problematic (backward compatibility). So, do you think developers will actually need these?

Reply:
Re-export could be skipped, however that's how we've already done things.

When I write jests tests, mocks, etc, I can import all the types which are used in a library, directly from that library (/lib/xp/name), rather than some from the list, and some from a type package (@enonic-types/root)

Reply:
If we stop re-exporting these types, the only place they’ll be used is in the core, right? Are you certain that these specific types need to be separate exports? We have plenty of other literal unions (like Content['workflow']['state'], Content['inherit'][number], Highlight['encoder'], etc.) that aren’t exported.

If you're using these types or believe others might need them, then it's fine to keep them. Otherwise, I’d prefer not to export them and not to have a dedicated type for them within the code.